composer.json

```diff
@@ -73,7 +73,8 @@
         "phpunit/phpunit": "^9.5",
         "symfony/browser-kit": "5.4.*",
         "symfony/css-selector": "5.4.*",
-        "symfony/phpunit-bridge": "5.4.*"
+        "symfony/phpunit-bridge": "5.4.*",
+        "vlucas/phpdotenv": "^5.6"
```

Comment:
@pspanja will additionally check if we can use Symfony Dotenv component instead of adding a new depependency.

Reply:
Updated in https://github.com/netgen/media-site/pull/263/commits/1c3d3053135e165c7ab2e17c3f156774117217bd